Makefile

```diff
@@ -52,9 +58,33 @@ else
   GLOBAL_CXXFLAGS += -O0 -U_FORTIFY_SOURCE
 endif
 
+include mk/platform.mk
+
+ifdef HOST_WINDOWS
+  # Windows DLLs are stricter about symbol visibilty than Unix shared
+  # objects --- see https://gcc.gnu.org/wiki/Visibility for details.
+  # This is a temporary sledghammer to export everything like on Unix,
```

Comment:
Yeah, adding exports for all symbols will probably be a bit of work

Reply:
Same. It's not an urgent todo.